### DISCUSSION TITLE
Update WebPack version for @bentley/webpack-tools-core

### DISCUSSION THREAD

**Discussion by PKASHYAP-BEC** [iTwin/itwinjs-core#2184] [General]
Hi team , 

is there any plan to update webpack for @bentley/webpack-tools-core package? if yes, when we can expect that.

**Comment by aruniverse** [comment]
Is there something specific you need from that package?

**Reply by PKASHYAP-BEC** [reply]
We want to upgrade it to webpack v 5.52 as we are getting some security issues in older version of webpack

**Reply by aruniverse** [reply]
We are on Webpack v4 and don't plan on moving to 5.x anytime soon. What's the security issue you're seeing? Maybe there's a fix in 4.x
